docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Modules d'extension et intégrations
```

Comment:
Non blocking, but noting that this title is quite long, causing it to wrap when it’s the current page:

<img width="277" alt="sidebar with page title" src="https://github.com/withastro/starlight/assets/357379/266e0d0b-de42-48a9-a164-d86a8a028024">

It’s an accurate translation though, so I’m happy if you are. I guess one other option could be “Modules d'extension et outils”?

Reply:
Great catch, and based on the content, your suggestion is totally valid so I went with it 👍